web/next.config.js

```diff
@@ -3,27 +3,133 @@ const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true',
 });
 const { PHASE_DEVELOPMENT_SERVER } = require('next/constants');
-const runtimeCaching = require('next-pwa/cache');
 
 const withPWA = require('next-pwa')({
   dest: 'public',
   runtimeCaching: [
     {
-      urlPattern: /\.(?:ts|m3u8)$/i,
```

Comment:
Why was this removed? Previously, the service worker was caching these without forcing it to pull from the network.

Reply:
I see, the original code used `NetworkOnly` handler which network request will always happen for this match, so I assumed the intention is to prevent ts and m3u8 being cached

How about I put this matching back using `CacheFirst` handler, so cached response will be used

Reply:
We don't ever want HLS assets to be cached, we want those requests to access the network every time. So `NetworkOnly` makes sense, we don't want `CacheFirst`.

Reply:
Thanks for clarification, I will do the changes accordingly